styles/charts.less

```diff
@@ -0,0 +1,147 @@
+.chart-tick {
```

Comment:
nitpick: in the future would you mind submitting two PR's, one for Webpack and one for styles? the merging of the two concerns is throwing me off a bit.